include/spdlog/details/logger_impl.h

```diff
@@ -64,7 +64,7 @@ inline void spdlog::logger::log(level::level_enum lvl, const char *fmt, const Ar
 #if defined(SPDLOG_FMT_PRINTF)
         fmt::printf(log_msg.raw, fmt, args...);
```

Comment:
Travis is failing on this line. I didn't test with any compile flags locally, which is why I didn't catch this. I genuinely don't know what the solution is because I don't know what the distinction between `printf` and `.write` was. I suspect we can/should just remove this branch, but I don't know.